.github/workflows/docs-deploy.yml

```
@@ -16,21 +16,21 @@ jobs:
   deploy-storybook:
     name: Deploy Storybook to Vercel
     runs-on: ubuntu-latest
-    if: false  # Disabled by default - requires VERCEL_TOKEN, VERCEL_ORG_ID, VERCEL_PROJECT_ID secrets
+    if: false # Disabled by default - requires VERCEL_TOKEN, VERCEL_ORG_ID, VERCEL_PROJECT_ID secrets
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)